test_vm_deployment/deploy_test_fullstack.sh

```diff
@@ -0,0 +1,62 @@
+#! /bin/bash
+
+. /home/zubhub/single_vm_deployment/onetime-setup.sh
+
+[ ! -d "/home/zubhub" ] && { echo >&2 "/home/zubhub dir not available.  Aborting."; exit 1; }
+[ ! -d "/home/zubhub_backend" ] && { echo >&2 "/home/zubhub_backend dir not available.  Aborting."; exit 1; }
+[ ! -d "/home/zubhub_frontend/zubhub" ] && { echo >&2 "/home/zubhub_frontend/zubhub dir not available.  Aborting."; exit 1; }
+
+echo "copying backend .env file"
+cp /home/zubhub_backend/.env /home/zubhub/zubhub_backend/.env
+echo "done copying backend .env file"
+
+echo "copying docker-compose file"
+cp /home/zubhub/test_vm_deployment/docker-compose.test.yml /home/docker-compose.test.yml
+echo "done copying docker-compose file"
+
+echo "removing old frontend folder"
+rm -rf /home/zubhub_frontend
+echo "removing old frontend folder"
+
+echo "remove old backend folder"
+rm -rf /home/zubhub_backend
+echo "removing old backend folder"
+
+echo "copying new frontend folder"
+cp -r /home/zubhub/zubhub_frontend/ /home/zubhub_frontend/
```

Comment:
Maybe we need to get a copy of the frontend `env` file too?
```suggestion
cp -r /home/zubhub/zubhub_frontend/ /home/zubhub_frontend/
cp -r /home/zubhub/zubhub_frontend/zubhub/.env.example /home/zubhub_frontend/zubhub/.env
```